test/ttt_test.exs

```diff
@@ -78,47 +78,47 @@ defmodule TTTTest do
   end
 
   test "Player tokens are updated and rendered on the board when user provides valid input" do
-    board = Board.setup
-    p1 = Player.human("Steven", :x)
-    p2 = Player.human("Connie", :o)
-    game = Game.setup(p1, p2)
-    {:ok, game} = Game.play_turn(game, :o, {0, 0})
-    {:ok, game} = Game.play_turn(game, :x, {2, 1})
-    {:ok, game} = Game.play_turn(game, :o, {1, 0})
-    {:ok, game} = Game.play_turn(game, :x, {1, 1})
-    {:ok, game} = Game.play_turn(game, :o, {2, 0})
+    board = Board.setup("", "")
+    p1 = Player.human({"Steven", "x"})
+    p2 = Player.human({"Connie", "o"})
+    game = Game.setup(p1, p2, 1)
+    {:ok, game} = Game.play_turn(game, p2, {0, 0})
+    {:ok, game} = Game.play_turn(game, p1, {2, 1})
+    {:ok, game} = Game.play_turn(game, p2, {1, 0})
+    {:ok, game} = Game.play_turn(game, p1, {1, 1})
+    {:ok, game} = Game.play_turn(game, p2, {2, 0})
 
     assert capture_io(fn -> TTT.update_visual(board, game) end) ==
-      "\n\n o | o | o\n---+---+---\n 4 | x | x\n---+---+---\n 7 | 8 | 9\n\n\n"
+      "\n\n o | o | o\n-----------\n 4 | x | x\n-----------\n 7 | 8 | 9\n\n\n"
   end
 
   test "Naive computer player will always take the next open spot following opponent's move if there are no preceding spots" do
-    p1 = Player.human("Steven", :x)
-    p2 = Player.naive_computer("Comp", :o)
-    game = Game.setup(p1, p2)
+    p1 = Player.human({"Steven", :x})
+    p2 = Player.naive_computer({"Comp", :o})
+    game = Game.setup(p1, p2, 1)
```

Comment:
I notice that you use the function name `setup` pretty often. Try to use a more descriptive name for what this function is doing. Maybe in this case it is `Game.new_game`?